### ISSUE TITLE
glsl-colormap

### ISSUE THREAD

**Issue by dfcreative** [bpostlethwaite/colormap#8]
We can borrow some ideas from [glsl-colormap](https://github.com/kbinani/glsl-colormap), pretty nice collection.

**Comment by bpostlethwaite** [comment]
Definitely worth porting them all over

**Comment by bpostlethwaite** [comment]
@dfcreative so lets meet up and figure out how to script those colormaps and port them over into the JSON blobs we have here

**Comment by dfcreative** [comment]
I can devote some time to it, that should not be difficult. Although @etpinard had an idea that maybe we better remove colormap from plotly dist, since we duplicate them as a constant.

**Comment by etpinard** [comment]
Can we just reuse https://github.com/plotly/plotly.js/blob/master/src/components/colorscale/make_color_scale_func.js

**Comment by bpostlethwaite** [comment]
@etpinard yep you only need the scales. I actually want to add this to the library independently from plotly.js. I didn't even realize plotly.js still used `colormap`

**Comment by etpinard** [comment]
Only gl3d `surface` and `mesh3d` uses colormap I believe. We should definitely keep it out of the regl2d rewrite.
